src/index.js

```diff
@@ -1,15 +1,37 @@
 import './style.css';
-import people from './modules/players.js';
-
-const playersScores = document.querySelector('.scores-list');
-
-playersScores.innerHTML = people
-  .map(
-    (player) => `
-    <tr>
-      <td>${player.name}</td>
-      <td>${player.score}</td>
-    </tr>
-  `,
-  )
-  .join('');
+import { createScore, getScore } from './modules/apiOperations.js';
+import Board from './modules/leadrBoard.js';
+
+const submit = document.getElementById('submit');
+
+submit.addEventListener('click', (e) => {
+  e.preventDefault();
+  const player = document.getElementById('player');
+  const score = document.getElementById('score');
+
+  createScore(player.value, score.value);
+  player.value = '';
+  score.value = '';
+});
+
+const players = new Board();
+const print = () => {
+  const result = getScore();
+  result.then((res) => {
+    players.data = res.result;
```

Comment:
- [ ]   You should completely use `async `and `await`  promises instead mixing it with `.then()` promise to consume data form the API.  `Async `and await help you to write a clean and minimized codes than .then().